schema/schema.go

```diff
@@ -161,20 +162,33 @@ func IsTableExist(conn mysql.Executer, schema string, name string) (bool, error)
 	return r.RowNumber() == 1, nil
 }
 
-func NewTable(conn mysql.Executer, schema string, name string) (*Table, error) {
+func NewTable(conn interface{}, schema string, name string) (*Table, error) {
```

Comment:
Thanks @shuhaowu 

But I don't like the interface here, the interface has no meaning here. If you want to support sql.DB, I prefer using another function like NewTableFromDB.